healthcareai/advanced_supvervised_model_trainer.py

```diff
@@ -72,34 +72,6 @@ def is_regression(self):
         """
         return self.model_type == 'regression'
 
-    def feature_scaling(self, columns_to_scale):
```

Comment:
I think you may have forgotten to commit the addition of this to transformers. :smile: